scRNA-seq-advanced/setup/glioblastoma/config.yaml

```diff
@@ -0,0 +1,7 @@
+# The main directory where files will be stored
+base_dir:  /shared/data/training-modules/scRNA-seq-advanced/data/glioblastoma
+mito_file: /shared/data/training-modules/scRNA-seq-advanced/data/reference/hs_mitochondrial_genes.tsv
```

Comment:
I don't see this `mito_file` used in the `Snakefile.` Is it being defined perhaps for future rules that will be written for processing?

Reply:
A relic from copying the PBMC config file...